src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
```

Comment:
Alright alright alright. There are more comments in this PR than actual lines of code, at which point I would name my variables as Britney Spears' titles if that could make you give me a :+1: :-)